tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Provide project scoped test fixtures."""
+import signal
+import subprocess
+import time
+import uuid
+
+import pytest
+
+from tests.common import ENGINE_MODULE, TOKEN, URL
+from tests.fake_client import TestClient
+
+
+@pytest.fixture(name="engine")
+def setup_engine():
+    """Set up engine."""
+    engine_args = f"python {ENGINE_MODULE} --debug --token {TOKEN}"
+    process = subprocess.Popen(engine_args.split())
+    time.sleep(2)  # This is needed to let the engine finish setup.
```

Comment:
It would be cool to accomplish this in another way, that doesn't rely in a specific time. But I couldn't think of anything simple that would do it.

If the engine setup time increases in the future this could break, so it's something we should try to enhance I think.

Reply:
We can start the client connection before the engine is up, the client is async, should be ok to wait until the engine is up.

Reply:
That's not working in my tests. I haven't really dug into it.